static/app/components/gridEditable/styles.tsx

```diff
@@ -79,8 +78,12 @@ export const Grid = styled('table')<{height?: string | number; scrollable?: bool
   margin: 0;
 
   z-index: ${Z_INDEX_GRID};
-  overflow-x: auto;
-  overflow-y: ${p => (p.scrollable ? 'scroll' : 'hidden')};
+  ${p =>
+    p.scrollable &&
+    `
+    overflow-x: auto;
+    overflow-y: scroll;
+    `}
```

Comment:
When `scrollable` is false, `overflow-y` should be set to `visible` (which is the default value), not `hidden`.